client/src/sessions/password.rs

```diff
@@ -0,0 +1,34 @@
+use super::session::Session;
```

Comment:
Nit: I guess I slightly prefer just doing `use crate::sessions::Session` instead of traveling up through `super`.